elm/sandbox/server/walker/walk_creator.py

```diff
@@ -34,38 +35,49 @@ def serialize_walker(self):
             joints.append((j[0], j[1]))
```

Comment:
here `j[0], j[1]` are also wrapped with numpy but it's only used to generate string instead of returning via flask endpoint. Either leaving them or changing into `.tiem()` is fine.